libraries/Microsoft.Bot.Connector/Microsoft.Bot.Connector.csproj

```diff
@@ -68,5 +68,14 @@
 		<ProjectReference Include="..\Microsoft.Bot.Schema\Microsoft.Bot.Schema.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
```

Comment:
Woah, what happened here? It seems like your VS went off the rails or something because it's hinting to the fallback version of the DLL instead of the NuGet package version. I would revert this and see what's going on locally to cause this.